### ISSUE TITLE
Automate check of docstrings

### ISSUE THREAD

**Issue by santisoler** [fatiando/community#150]
**Description:**

Since we are already using the numpydoc style in our docstrings, it would be nice if we automate the process of checking that they are properly written.

This could be done in the past using [`pydocstyle`](https://github.com/pycqa/pydocstyle) (and the [`flake8-docstrings`](https://github.com/pycqa/flake8-docstrings) plugin). But `pydocstyle` has been deprecated, and they recommend using `ruff` instead.

`ruff` is not only checking docstrings, but a whole linter (kind of an alternative to flake8). I think it would be nice to have this feature, but that would require us to discuss a little bit about depending on `ruff`.

Related to #149. We should tackle this after we merge #149.

**Apply to:**

- [ ] [boule](https://github.com/fatiando/boule)
- [ ] [harmonica](https://github.com/fatiando/harmonica)
- [ ] [pooch](https://github.com/fatiando/pooch)
- [ ] [verde](https://github.com/fatiando/verde)
- [ ] [ensaio](https://github.com/fatiando/ensaio)
- [ ] [choclo](https://github.com/fatiando/choclo)
- [ ] [dependente](https://github.com/fatiando/dependente)
- [ ] [burocrata](https://github.com/fatiando/burocrata)
- [ ] [website](https://github.com/fatiando/website)
- [ ] [tutorials](https://github.com/fatiando/tutorials)
- [ ] [community](https://github.com/fatiando/community)
- [ ] [.github](https://github.com/fatiando/.github)

**Further instructions:**

* Start by opening Pull Requests on each repository listed above.
* Optionally, we can open Issues on each repository if further discussion specific to that repository is needed.
* Mention this Issue on every Issue or Pull Request opened on each opened: `Related to fatiando/community#XX`
* Check-off the repository on the list above once the Pull Request is merged.
* Close this issue when all items are checked-off.

**We want your help!**

We know that maintenance tasks are very demanding, so we don't expect a single person to tackle this issue by themselves. Any help is very welcomed, so please comment below that you want to take care of the changes on any repository and we will assign it to you.
